pom.xml

```diff
@@ -115,15 +115,15 @@ Arrays, Maps or Strings. The iteration protocol was inspired by the Smalltalk co
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <slf4j.version>2.0.7</slf4j.version>
+        <slf4j.version>2.0.16</slf4j.version>
         <clover.version>4.4.1</clover.version>
-        <scala.version>2.12.19</scala.version>
+        <scala.version>2.13.15</scala.version>
```

Comment:
I don't have a problem with this but I thought I tried upgrading Scala past 2.12 in the past and IIRC there were breaks?

Reply:
the build passes?